content/blog/a-summary-of-the-july-retrospectives/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: A Summary of the July Retrospectives
+date: "2022-08-24T12:20:00+00:00"
+author: shelleylambert
+description: This post summarizes the retrospective assessment of the July 2022 CPU and various respins thereafter.
+tags:
+  - Temurin
+  - AQAvit
+---
+
+## Introduction 
+As most of our large consumer base already knows, Eclipse Adoptium is *the* community 'build, distribution and verification' project for the upstream OpenJDK project.  We serve up millions of binaries per week, as can be seen in the download trends in our dashboard (see [dash.adoptium.net](https://dash.adoptium.net/)).  We deliver these binaries in the most transparent manner.  There is no 'mystery meat' in our recipe.  In the spirit of such great transparency, this post serves to share our insights of the post mortem analysis of our most recent release activities.
+
+We follow a strict regime of continuous improvement at the project, which means after each of our release cycles, conducting thorough retrospectives on what went well, what did not go well, and what actions are needed to improve in the future.  
+
+## Goals and Metrics
+All good retrospective activities start with "what were we trying to achieve"?  During the July CPU, we were targeting to deliver 42 different 'products', which is essentially the platform/version combinations of Eclipse Temurin that we build, test and distribute at the project.
+
+Based on download numbers, we have divided these products into 'primary' and 'secondary' platforms.  Currently, x64 Linux, x64 Windows and x64 Mac are the leading downloaded platforms, so they are categorized as 'primary' platforms.  The remaining platforms are categorized as 'secondary' platforms.  Within the secondary platforms, some may even be considered 'best effort' platforms which as the name suggests are put at the bottom of the priority list.  To keep our goals simple, we aim to release the primary platforms within 2 days of the OpenJDK GA tag being available.  For secondary platforms, our target is to release within 7 days of the GA tag availability.  These are summarized in a release status issue to communicate on-going status to the community.  The July release activities were tracked in [adoptium/issues/153](https://github.com/adoptium/adoptium/issues/153). 
```

Comment:
Maybe add something here relating to "during our release cycles we frequently get asked about people's favourite platforms and when they are going to get released" and how we are also concerned about time to release. (I'm thinking splitting this paragraph at "Too keep our goals simple" and adding that information somewhere between the two / at the start of the second, but I think the "We listen to your feedback" on this message is good to convey in this blog.